### ISSUE TITLE
Use forkeable document format, not PDF

### ISSUE THREAD

**Issue by gagarine** [DP-3T/documents#36]
PDF is not very cool on GitHub.

**Comment by bdsl** [comment]
I think it's probably fine - these are documents for humans to read, not computer software - essentially academic papers like you might find on a preprint repository. You can't just change a line, run the tests and assume it's good.

As long as the people working on the DP-3T team can update the documents easily that may be enough. Anyone independent who wants to publish something based on these documents but different probably should follow academic norms, and write their own documents using appropriate referencing when they take ideas from these ones.

**Comment by gagarine** [comment]
I'm looking forward to having to compare the next version line by line to find the change instead of looking at a diff.

**Comment by bdsl** [comment]
I agree there would be real advantages to using a plain-text easily diff-able format, e.g. Markdown or LaTex. I suppose my comment was just to say that might not outweigh the benefits of whatever system the authors have used so far to create the PDFs, especially that's something they are familiar and efficient with.

**Comment by sebastianbuettrich** [comment]
Not only are the docs in PDF, they are in image based PDF.
This is indeed not very well suited for collaborative work on text documents (can't copy/quote/fork)
and should be easy to change, as authors have a text version at hand, before saving as PDF.

**Comment by gagarine** [comment]
@sebastianbuettrich it's because you see the PDF preview from Github. If you download it it's fine.

@bdsl  As a side note, those PDF were probably created from Google Drive (produced by Skia/PDF m83).

**Comment by sebastianbuettrich** [comment]
@gagarine - ah! silly me ... not used to seeing PDFs in repos ..;) - thanks.

**Comment by s-chtl** [comment]
Hi everyone, thank you for reaching out. For now we have opted for a changelog at the beginning of the modified docs to keep track of the versioning.  
I will close the issue for now.

**Comment by LilithWittmann** [comment]
That's absolutely not the same and does not encourage collaboration at all.

**Comment by gagarine** [comment]
Fake open source.

**Comment by s-chtl** [comment]
Hi everyone, sorry if my previous post was confusing. Please be assured that we have understood your concern about pdf. I took the liberty to mark the issue as closed for now to help us triage a bit more the threads. I'll keep it open then so that people know this issue has been raised before and avoid confusion.

**Comment by kollokollo** [comment]
Maybe in addition to markup or LaTex format, I could recommend asciidoc. Similar to markup, It renders nicely in github and also can produce nice looking printable (.pdf) documents.
